src/apiCalls.js

```diff
@@ -1,14 +1,10 @@
 
-export const getPlaces = (url) => {
-    return fetch(url)
+export function getPlaces () {
+    return fetch('https://pet-furriendly-server.onrender.com/api/v1/places')
```

Comment:
What was the thought behind removing the URL parameter and hardcoding our server endpoint into the function instead?